tasks/TaskTypes.py

```diff
@@ -20,3 +20,6 @@ class TaskType(enum.Enum):
     SENTIMENT_ANALYSIS = (
         12,
     )  # This is a specific type of text classification with unique properties
+    PARAPHRASE_DETECTION = (13,)
+    TEXTUAL_ENTAILMENT = (14,)
+
```

Comment:
I think quality estimation can be another task of SentencePairOperation, where the inputs are source and translation and the output is a continuous score.

Reply:
Excellent idea! I will add it!